connect/file/src/main/java/org/apache/kafka/connect/file/FileStreamSourceConnector.java

```diff
@@ -95,4 +96,18 @@ public void stop() {
     public ConfigDef config() {
         return CONFIG_DEF;
     }
+
+    @Override
+    public ExactlyOnceSupport exactlyOnceSupport(Map<String, String> props) {
```

Comment:
Should we also document this in the [Connector Development Guide](https://kafka.apache.org/documentation/#connect_development) (along with the other new `SourceConnector::canDefineTransactionBoundaries` method)?

Reply:
I have plans to add a dedicated section for exactly-once support to our docs; I think it'd be better to put that information there since it's a fairly-advanced concept and probably wouldn't be too useful for people just getting up to speed with the Connector API. Thoughts?

Reply:
Thanks Chris, I think that makes sense.